app/main.py

```diff
@@ -1,3 +1,62 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        if isinstance(other, int):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        if isinstance(other, int):
+            self.km += other
+            return self
+
+    def __mul__(self, number):
+        return Distance(self.km * number)
+
+    def __truediv__(self, number):
+        return Distance(round(self.km / number, 2))
+
+    def __lt__(self, other):
+        if isinstance(other, Distance):
+            return True if self.km < other.km else False
```

Comment:
```suggestion
            return self.km < other.km
```
The redundant ternary operator